components/webfield/SeniorAreaChairConsole/PaperStatus.js

```diff
@@ -46,6 +47,23 @@ const PaperRow = ({ rowData, selectedNoteIds, setSelectedNoteIds, decision, venu
       seniorAreaChairName
     )} Console](/group?id=${venueId}/${seniorAreaChairName}#${submissionName}-status)`
   )
+  const getManualAssignmentUrl = (role) => {
+    if (!assignmentUrls) return null
+    const assignmentUrl = assignmentUrls[role]?.manualAssignmentUrl // same for auto and manual
+    // auto
+    const isAssignmentConfigDeployed = sacConsoleData.invitations?.some(
+      (p) => p.id === `${venueId}/${role}/-/Assignment`
```

Comment:
this shouldn't be hardcoded, we need to read the invitation id from the console properties